src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -122,7 +122,14 @@ public class PublicationService : IPublicationService
             return await _persistenceHelper
                 .CheckEntityExists<Publication>(publicationId)
                 .OnSuccess(_userService.CheckCanUpdatePublication)
-                .OnSuccessDo(_ => ValidateSelectedTopic(updatedPublication.TopicId))
+                .OnSuccessDo(async publication =>
+                {
+                    if (publication.TopicId != updatedPublication.TopicId)
```

Comment:
Firstly the method allows the Topic to be changed because it's been designed  as quite a generic RESTful method where the update is allowing modification on any field of the Publication including what topic it belongs to. I'm not sure of the history of it or if I necessarily agree with the way it's being used. We should discuss this approach.

It's used by the "Manage themes and topics" where the topic can be changed as well as the "Manage Publication" where in theory the topic shouldn't be changed.

Changing the topic checks the user has permission to create a new Publication in the topic its being moved into, using `CheckCanCreatePublicationForTopic` which Publication Owner won't have permission for, just like a Publication Owner can't use the "Manage themes and topics" functionality. Only BAU has permission to use that.

The change I'm making here is to prevent checking that permission if it's not changed.